components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -176,10 +173,13 @@ def custom_index(request, conn=None, **kwargs):
 ##############################################################################
 # views
 
+# from omeroweb.webgateway import LoginView
```

Comment:
Fixed. Seems that github doesn't automatically hide comments in 'Reviews' when lines are edited.